subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -58,6 +58,7 @@ config BT_FAST_PAIR_ADVERTISING
 	default y
 	select BT_PERIPHERAL
 	select BT_PRIVACY
+	select ENTROPY_GENERATOR
```

Comment:
Please add the same to `BT_FAST_PAIR_GATT_SERVICE`. It also uses the `sys_csrand_get` API